CMSIS/NN/Include/arm_nn_types.h

```diff
@@ -88,6 +88,19 @@ typedef struct
 
 } cmsis_nn_conv_params;
 
+/** CMSIS-NN object for Depthwise convolution layer parameters */
+typedef struct
+{
+    int32_t             input_offset;   /**< Zero value for the input tensor */
+    int32_t             output_offset;  /**< Zero value for the output tensor */
+    int32_t             ch_mult;        /**< Channel Multiplier. ch_mult * in_ch = out_ch */
+    cmsis_nn_tile       stride;
+    cmsis_nn_tile       padding;
+    cmsis_nn_tile       dilation;
+    cmsis_nn_activation activation;
+
```

Comment:
I am afraid you need to remove this empty line to avoid merge conflicts

Reply:
I'll get https://github.com/ARM-software/CMSIS_5/pull/930 in first, rebase and then update this..